react-common/components/controls/Button.tsx

```diff
@@ -101,13 +102,15 @@ export const Button = (props: ButtonProps) => {
             aria-setsize={ariaSetSize}
             aria-describedby={ariaDescribedBy}
             aria-selected={ariaSelected}>
-                <span className="common-button-flex">
-                    {leftIcon && <i className={leftIcon} aria-hidden={true}/>}
-                    <span className="common-button-label">
-                        {label}
-                    </span>
-                    {rightIcon && <i className={"right " + rightIcon} aria-hidden={true}/>}
-                </span>
+                {(leftIcon || rightIcon || label) && (
```

Comment:
I'm not sure why this is needed. Since we check for the left and right icon within the span, the check before the span feels a bit much since the label logic hasn't changed.

Reply:
I can embed the label check.

Reply:
Oh I recall now, this was to avoid rendering the outer "common-button-flex" div container if none of the inner components would render.

Reply:
I see. So there's a possibility that label could be nothing?

Reply:
Yes, label is an optional prop.